crates/runtime/benches/bench_s3/mod.rs

```diff
@@ -98,9 +98,7 @@ fn make_dataset(path: &str, name: &str) -> Dataset {
     dataset
 }
 
-fn get_test_queries(acceleration: &Option<Acceleration>) -> Vec<(&'static str, &'static str)> {
-    let is_accelerated = acceleration.is_some();
-
+fn get_test_queries(_acceleration: &Option<Acceleration>) -> Vec<(&'static str, &'static str)> {
```

Comment:
Not removing this parameter so we can re-use it in the future when we have more tests, I expect at least few of them will require extra work for specific acceleration engine.